test/test.bugsnag.js

```diff
@@ -540,6 +540,17 @@ describe("Bugsnag", function () {
         // Confirm we kept the most recent 20 breadcrumbs instead of the first 20
         assert.equal(requestData().params.breadcrumbs[19].metaData.message, "I am breadcrumb 20");
       });
+
+      it("lets me set metaData with self nesting", function() {
+        var metaData = {a: metaData, b: "text"};
```

Comment:
This sets `metaData.a` to `undefined`. To make it recursively point to itself, you'd need to do something like

```javascript
var metaData = {a: "text"};
metaData.b = metaData;
```